### DISCUSSION TITLE
Fundamentos do blazor com .net 8 - Como definir a pagina de inicio?

### DISCUSSION THREAD

**Discussion by dalmiroe** [balta-io/forum#482] [Tira Dúvidas] [answered]
Bom dia. Gostaria de fazer um teste que é o seguinte. 
Colocar uma pagina de entrada e depois chamar a pagina com o menu lateral chamando as demais.
Fiz o seguinte. Criei um layout na pasta leyout chamada MainLayoutIndex so com @body e criei uma pagina index informando o layout novo tb. 
Como tenho que fazer para chamar o index.razor ?
Obrigado

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @dalmiroe, como vai?

A página de início seria o arquivo `.razor` com a definição `@page "/"`, então se quer que seu `índex.razor` seja a primeira página a ser mostrada, basta adicionar (Ou alterar) esta linha no início do arquivo.

Em adicional, recomendo seguir todas as carreiras do site, nelas falamos como o ASP.NET e as rotas funcionam:
https://balta.io/carreiras

Lembro também que o curso de Fundamentos do Blazor faz uso de conceitos abordados nos outros cursos da carreira de Blazor, que são indispensáveis para o aproveitamento do mesmo:
https://balta.io/carreiras/aspnet-blazor-developer

**Comment by dalmiroe** [comment]
Obrigado, vou testar....obs estou fazendo todos os cursos, a intenção é completar todas as carreiras disponíveis
